handlers/register_user_handler.go

```diff
@@ -1,19 +1,33 @@
 package handlers
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"net/smtp"
+	"strings"
 
 	"user_management/utils"
+
+	"github.com/jordan-wright/email"
+	"github.com/knadh/koanf/v2"
+	"github.com/redis/go-redis/v9"
 )
 
 type RegisterRequest struct {
 	Displayname string `json:"displayname"`
 	Email       string `json:"email"`
 }
 
-func RegisterUserHandler(w http.ResponseWriter, r *http.Request) {
+type WelcomeTemplateData struct {
+	Displayname string
+	SignupUrl   string
+}
+
+var REGISTER_PAGE_URL = "http://localhost:8080/api/v1/users/register"
```

Comment:
Can you set the base URL ```http://localhost:8080/``` as a global variable in main. You can pass the baseUrl as an input to the handler when it gets called in ```main.go```.

Reply:
I set http://localhost:8080/ since I didn't know what our hostname will be called. Is it still going to be localhost in prod? I'm not sure since we're using Docker.

Reply:
Let's put that in the config toml.